src/global/helpers/composed-path-has-attribute.ts

```diff
@@ -10,7 +10,7 @@ export function composedPathHasAttribute(
   event: Event,
   attribute: string,
   endElement = null
-): EventTarget {
+): HTMLElement {
```

Comment:
Sorry, hopefully last comment, but it can also be null which should be reflected in types